src/utils/Filter.ts

```diff
@@ -0,0 +1,4 @@
+export enum Filter {
+  MALE = 'm',
+  FEMALE = 'f',
```

Comment:
```suggestion
export enum Filter {
  Male = 'm',
  Female = 'f',
```
Enum properties should be in PascalCase, not upper case